src/sysmem.h

```diff
@@ -33,6 +33,9 @@
 */
 extern Int SyStorMax;
 extern Int SyStorOverrun;
+// A constant which can be assigned to SyStorOverrun to denote the overrun
+// has already been printed.
+enum { SY_STOR_OVERRUN_HANDLED = 999 };
```

Comment:
So, `SyStorOverrun` has three valid values: 0, -1 and 999. For one of them, an enum is defined. If you already go for an enum (a good idea), why not define all allowed values, and then turn `SyStorOverrun` into a variable of that enum type? I.e.
```
enum StorOverrunType { ... };
extern enum StorOverrunType SyStorOverrun;
```

Reply:
Yes, this generally needs cleaning up. This was just the start of a hack when I realised I needed a way of telling SyInterrupt that I'd already handled the memory overflow (as I thought that was the easiest way of handling it). It might be I can just clear it instead, then no need this.